codalab/apps/teams/models.py

```diff
@@ -283,9 +283,31 @@ def __unicode__(self):
 
 
 class TeamMembership(models.Model):
+    user = models.ForeignKey(settings.AUTH_USER_MODEL)
+    team = models.ForeignKey(Team)
+    is_invitation = models.BooleanField(default=False)
+    is_request = models.BooleanField(default=False)
+    start_date = models.DateTimeField(null=True, blank=True, verbose_name="Start Date (UTC)")
+    end_date = models.DateTimeField(null=True, blank=True, verbose_name="End Date (UTC)")
```

Comment:
Ah, we made this `end_date` attribute?

Reply:
I don't believe so? I think I moved the fields above the meta declaration be styled like the rest of the classes.

Reply:
gotchya